### PR TITLE
Update Makefile dependencies/build targets

### PR DESCRIPTION
**Pull request by dblandin**
- Install dev dependencies within pyenv
- Run pyinstaller within pyenv

Co-authored-by: @izakp 

I think this might supersede https://github.com/artsy/hokusai/pull/193 ?

### CODE REVIEW

**Comment by izakp** [comment]
Thanks @dblandin - I think we still need to `pip install .` in the Dockerfile outside of a Pipenv so it doesn't have to be activated every time the container starts

**Comment by dblandin** [comment]
> I think we still need to pip install . in the Dockerfile outside of a Pipenv so it doesn't have to be activated every time the container starts

@izakp Made a couple more adjustments. Let me know what you think!

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+RUN curl -L https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v0.4.0/aws-iam-authenticator_0.4.0_linux_amd64 -o /usr/local/bin/aws-iam-authenticator && \
+      chmod a+x /usr/local/bin/aws-iam-authenticator
 
-ADD . /src
-WORKDIR /src
+COPY . /src
 
 # Install Hokusai
-RUN pipenv install --deploy
+RUN pip install .
```
Comment (izakp) [inline Dockerfile:24]: 👍

**Comment by izakp** [comment]
Thanks this looks good to me!

**Comment by izakp** [comment]
Darn master build failed - will have a look

**Pull request merged**